bookShop/src/app/shared/shared.module.ts

```diff
@@ -5,15 +5,21 @@ import { BrowserModule } from '@angular/platform-browser';
 import { OrdersModule } from './orders.module';
 import { ChangeBackgroundDirective } from './directives/change-background.directive';
 import { ChangeOnclickElementDirective } from './directives/change-onclick-element.directive';
+import { OrderByPipe } from './pipes/order-by.pipe';
 
 @NgModule({
-  declarations: [ChangeBackgroundDirective, ChangeOnclickElementDirective],
+  declarations: [
+    ChangeBackgroundDirective,
+    ChangeOnclickElementDirective,
+    OrderByPipe,
+  ],
   imports: [BrowserModule, FormsModule],
```

Comment:
I think you should add `CommonModule,  OrdersModule` here as well like in exports module.

Reply:
Corrected